### PR TITLE
Move to rollup

### PR DESCRIPTION
**Pull request by thabarbados**
### Fix or Enhancement?
Remove webpack, add rollup

- [x] All tests passed

### Environment
- OS: BigSur MacOs
- NPM Version: 6.14.15

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,13 +47,12 @@
     "types/*.d.ts"
   ],
   "scripts": {
-    "build": "yarn run release && yarn run build:es",
-    "build:es": "cross-env BABEL_ENV=es babel src --out-dir es",
+    "build": "yarn release && rimraf dist && rollup -c",
```
Comment (dangreen) [inline package.json:50]:
```suggestion
    "emitDeclarations": "tsc --skipLibCheck --emitDeclarationOnly",
    "build": "rollup -c & yarn emitDeclarations",
```

File: package.json [hunk 2]
```diff
@@ -83,56 +81,39 @@
...
     "jest-canvas-mock": "^2.3.1",
-    "json-loader": "^0.5.4",
     "lint-staged": "^12.1.2",
-    "opn": "^5.1.0",
-    "portfinder": "^1.0.13",
     "prettier": "2.5.1",
-    "shelljs": "^0.7.7",
+    "rimraf": "^3.0.2",
+    "rollup": "^2.66.1",
+    "rollup-plugin-swc": "^0.2.0",
```
Comment (dangreen) [inline package.json:107]: `@swc/core` is peer dependency, where is it?

alos `@swc/helpers` ?

**Review by dangreen — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (thabarbados) [reply package.json]: fix

File: package.json (hunk 2 above)
Comment (thabarbados) [reply package.json:109]: `@swc/core` was added in  https://github.com/apertureless/vue-chartjs/pull/733 (https://github.com/apertureless/vue-chartjs/pull/733/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R77)   
add `@swc/helpers`

File: package.json [hunk 3]
```diff
@@ -48,13 +47,13 @@
     "types/*.d.ts"
   ],
   "scripts": {
-    "build": "yarn run release && yarn run build:es",
-    "build:es": "cross-env BABEL_ENV=es babel src --out-dir es",
+    "emitDeclarations": "tsc --skipLibCheck --emitDeclarationOnly",
+    "build": "rimraf dist && rollup -c & yarn emitDeclarations",
```
Comment (dangreen) [inline package.json:51]: Please remove rimraf here - it doesn't necessary

**Review by dangreen — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (thabarbados) [reply package.json]: removed rimraf

**Pull request merged**